openid-sharedsignals-framework-1_0.md

```diff
@@ -1373,7 +1434,7 @@ stream_id
 
 > REQUIRED. The stream whose status is being queried.
 
-subject
+sub_id
```

Comment:
I don't know if I like changing these arguments as part of this pull request. `subject` is clearer and there is no requirement that the argument name to this function matches the claim name in the SET that get produced by this Transmitter.